MapboxDirections/MBRouteLeg.swift

```diff
@@ -98,6 +134,26 @@ open class RouteLeg: NSObject, NSSecureCoding {
      */
     open let steps: [RouteStep]
     
+    /**
+     An array of distances representing the distances between nodes.
+     */
+    open let segmentDistances: [CLLocationDistance]?
+    
+    /**
+     An array of expected travel times for the space between each geometry node. This value is dynamic and accounts for any and all conditions that may change along current segment.
+     */
+    open let expectedSegmentTravelTimes: [TimeInterval]?
+    
+    /**
+     An array of current speeds along segment. This value is dynamic and does not represent the speed limit but rather the average speed for a given segment
```

Comment:
> An array containing the expected average speed (measured in meters per second) between each coordinate in the route leg geometry.
> 
> These values are dynamic; rather than speed limits, they account for the road’s classification and/or any traffic congestion (if the profile identifier is set to `.automobileAvoidingTraffic`).
> 
> This property is set if the `RouteOptions.attributeOptions` property contains `.speed`.